__tests__/app.test.js

```diff
@@ -199,4 +199,100 @@ describe('GET api/articles/:article_id/comments', () => {
                 expect(body.msg).toBe("bad request")
             })
     })
-});
\ No newline at end of file
+});
+
+describe('POST api/articles/:article_id/comments', () => {
+    test(`sending a POST request to api/articles/:articles_id/comments with a username found within the users table successfully adds a comment to the comment table, returns a status 200, and sends back an object with the new comment added as a key value, on the key of comment.`, () => {
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send({ 
+            username: "rogersop", 
+            body : "spaceships dont come equipped with rear view mirrors"
+            })
+            .expect(200)
+            .then(({body}) => {
+                const comment = body.comment
+                expect(comment).toMatchObject({
+                    "comment_id" : 19,
+                    "votes" : 0,
+                    "created_at" : expect.any(String),
+                    "author" : "rogersop",
+                    "body" : "spaceships dont come equipped with rear view mirrors",
+                    "article_id" : 1
+                })
+            })          
+    })
+    test(`sending a POST request to api/articles/:articles_id/comments with a username found within the users table, but an article_id that doesnt exist returns a 400 with message "bad request".`, () => {
+        return request(app)
+            .post('/api/articles/5000/comments')
+            .send({ 
+                username: "rogersop", 
+                body : "be responsible for what you say because the words you speak can truly cause dismay"
+            })
+            .expect(400)
```

Comment:
this one should be a 404? it's a valid request but that article cannot be found.